.build/default.build.settings

```diff
@@ -184,14 +184,14 @@
 
-  <property name="app.metrics" value="${dirs.program_files}${path.separator}Microsoft Visual Studio 10.0${path.separator}Team Tools${path.separator}Static Analysis Tools${path.separator}FxCop${path.separator}Metrics.exe" overwrite="false" />
-  <property name="app.metrics" value="${dirs.program_files_x86}${path.separator}Microsoft Visual Studio 10.0${path.separator}Team Tools${path.separator}Static Analysis Tools${path.separator}FxCop${path.separator}Metrics.exe" if="${not file::exists(app.metrics)}" />
+  <property name="app.metrics" value="${dirs.program_files}${path.separator}Microsoft Visual Studio 11.0${path.separator}Team Tools${path.separator}Static Analysis Tools${path.separator}FxCop${path.separator}Metrics.exe" overwrite="false" />
+  <property name="app.metrics" value="${dirs.program_files_x86}${path.separator}Microsoft Visual Studio 11.0${path.separator}Team Tools${path.separator}Static Analysis Tools${path.separator}FxCop${path.separator}Metrics.exe" if="${not file::exists(app.metrics)}" />
   <property name="run.metrics" value="${file::exists(app.metrics)}" overwrite="false" />
 
-  <property name="app.fxcop" value="${path::get-full-path(dirs.program_files)}${path.separator}Microsoft Visual Studio 10.0${path.separator}Team Tools${path.separator}Static Analysis Tools${path.separator}FxCop${path.separator}FxCopCmd.exe" overwrite="false" />
-  <property name="app.fxcop" value="${path::get-full-path(dirs.program_files_x86)}${path.separator}Microsoft Visual Studio 10.0${path.separator}Team Tools${path.separator}Static Analysis Tools${path.separator}FxCop${path.separator}FxCopCmd.exe" if="${not file::exists(app.fxcop)}" />
-  <property name="app.fxcop" value="${path::get-full-path(dirs.program_files)}${path.separator}Microsoft Fxcop 10.0${path.separator}FxCopCmd.exe" if="${not file::exists(app.fxcop)}" />
-  <property name="app.fxcop" value="${path::get-full-path(dirs.program_files_x86)}${path.separator}Microsoft Fxcop 10.0${path.separator}FxCopCmd.exe" if="${not file::exists(app.fxcop)}" />
+  <property name="app.fxcop" value="${path::get-full-path(dirs.program_files)}${path.separator}Microsoft Visual Studio 11.0${path.separator}Team Tools${path.separator}Static Analysis Tools${path.separator}FxCop${path.separator}FxCopCmd.exe" overwrite="false" />
+  <property name="app.fxcop" value="${path::get-full-path(dirs.program_files_x86)}${path.separator}Microsoft Visual Studio 11.0${path.separator}Team Tools${path.separator}Static Analysis Tools${path.separator}FxCop${path.separator}FxCopCmd.exe" if="${not file::exists(app.fxcop)}" />
+  <property name="app.fxcop" value="${path::get-full-path(dirs.program_files)}${path.separator}Microsoft Fxcop 11.0${path.separator}FxCopCmd.exe" if="${not file::exists(app.fxcop)}" />
+  <property name="app.fxcop" value="${path::get-full-path(dirs.program_files_x86)}${path.separator}Microsoft Fxcop 11.0${path.separator}FxCopCmd.exe" if="${not file::exists(app.fxcop)}" />

```

Comment:
Okay dokay, that makes sense.  I did the above just to get it to work.  Will add in the additional entries. :+1: